compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/codegen/ExpressionCodegen.kt

```diff
@@ -134,10 +135,24 @@ class ExpressionCodegen(
                 mv.areturn(returnType)
             }
         }
-        writeLocalVariablesInTable(info)
+        val endLabel = writeLocalVariablesInTable(info)
+        createLocalVariablesForParameters(startLabel, endLabel)
```

Comment:
It would be cleaner to create new label here

Reply:
OK